README.md

```diff
@@ -5,38 +5,45 @@ Coming to you live.
 ## Setup
 
 Install a rust toolchain with [rustup.rs](https://rustup.rs):
+
 ```sh
 curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
 cargo --version
 rustc --version
 ```
 
 Clone the repo:
+
 ```sh
 git clone https://github.com/foltik/lsd
 cd lsd
 ```
 
+Create a .env file with `DATABASE_URL=sqlite://db.sqlite`
+
 Initialize dev database:
+
 ```sh
+cargo install sqlx-cli --no-default-features --features sqlite
 cargo sqlx database create
+cargo sql migrate run
```

Comment:
These two commands can be replaced with `cargo sqlx database setup` (which also creates the database file itself)

Reply:
(either way, typo here: `cargo sql` -> `cargo sqlx`)

Reply:
Sorry I should've been more clear! *only* `cargo sqlx database setup` is needed, not `create` too.